packages/mobile-top-nav/src/media-subnav.js

```diff
@@ -0,0 +1,70 @@
+import { LitElement, html } from 'lit-element';
+import menus from './data/menus';
+import './wayback-search';
+import './more-slider';
+import subnavCss from './css/media-subnav';
+
+class MediaSubnav extends LitElement {
+  constructor() {
+    super();
+    const defaultLinks = { iconLinks: [], featuredLinks: [], links: [] };
+
+    // Begin properties not monitored by LitElement
+    this.links = menus[this.menu] || defaultLinks;
+    this.templates = {
+      web: () => html`<wayback-search></wayback-search>`,
+      more: () => html`<more-slider></more-slider>`,
+    };
+  }
+
+  static get properties() {
+    return {
+      menu: { type: String }
+    };
+  }
+
+  shouldUpdate() {
+    if (menus[this.menu]) {
+      this.links = menus[this.menu];
```

Comment:
Q: why are we using a lifecycle method here if we aren't using the `changedProperties` arg that this fn provides? I am inclined to think that we can do this at the render level?